example/macos/.gitignore

```diff
@@ -4,3 +4,7 @@ xcuserdata
 Flutter/ephemeral/
 # Created by CocoaPods for plugins.
 Pods/
+
+# Create by flutter tool for XCode
+FlutterInputs.xcfilelist
```

Comment:
I didn't really know where to put these files so I stuck them in the root.

Reply:
If they are created by `flutter`, and not intended to be checked in, they should be in Flutter/ephemeral.

Reply:
File paths moved, still need to update tool pr